Makefile

```
@@ -255,7 +258,7 @@ watch: all
 ifneq ($(MAIN_TARGET),)
 .PHONY: spellcheck
 spellcheck:
-	for i in $$(find . -name \*.tex); do aspell check --mode=tex --personal=$(shell pwd)/.aspelldict "$$i" ; done
+	for i in $$(find . -name \*.tex $(foreach exclude,$(SPELLCHECK_EXCLUDES),! -name $(exclude))); do aspell check --mode=tex --personal=$(shell pwd)/.aspelldict "$$i" ; done
```

`find`'s `-name` argument won't actually handle situations like "exclude `foo/bar.tex`". For that, you need to use `-path`. Also, some shells sometimes do weird things with `!`, so it helps to put it into an explicit string (I am not sure it is necessary in this case, but doesn't hurt to put it into a "don't interpret this" string).
```suggestion
	for i in $$(find . -name \*.tex $(foreach exclude,$(SPELLCHECK_EXCLUDES),'!' -path './$(exclude)')); do aspell check --mode=tex --personal=$(shell pwd)/.aspelldict "$$i" ; done
```